packages/terra-alert/src/Alert.module.scss

```diff
@@ -42,7 +42,7 @@
 
   .body-std {
     margin-bottom: 0.7143rem;
-    margin-right: 0;
+    margin-right: 0.6667rem;
```

Comment:
I'm surprised this didn't fail any wdio tests, the text must not extend all the way to the edge in any of the test examples

Reply:
Should we add a test that would fail if this changed?

Reply:
Added a test case which would shift the letters in the alert around if padding is adjusted. https://github.com/cerner/terra-core/pull/1959/commits/b79aa2e7f86e47908b37e6cdf93eeb179f001545